README.md

```diff
@@ -52,11 +53,20 @@ Kalispell Software Crafters prioritizes marginalized people's safety over privil
 * Communicating in a 'tone' you don't find congenial.
 * Criticizing racist, sexist, cissexist, or otherwise oppressive behavior or assumptions.
 
+The above sections focus on harassment. The following section describes other actions that are not allowed within our communities or at our events.
+
+* Solicitation
+* Advertisements
+* Excessive strong language (curse words/swearing)
+* Posting spam to our online communities including the Meetup page
+
+In general do not use obscene, lewd, lascivious, filthy, or excessively violent language. Do not harass, insult, taunt, provoke, demean, or personally attack other members. Be friendly even if others are not.
```

Comment:
I pulled this line from Mozilla's forum rules and guidelines (see the language and conduct section): https://support.mozilla.org/en-US/kb/forum-rules-and-guidelines#w_language-and-conduct